source/testing/unit-testing-basics.md

```diff
@@ -33,25 +33,51 @@ should trigger the computed property), and assert that the logic in our
 computed property is working correctly.
 
 ```tests/unit/models/some-thing-test.js
-import { moduleFor, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 
-moduleFor('model:some-thing', 'Unit | some thing', {
-  unit: true
-});
+import { run } from '@ember/runloop';
+
+module('Unit | Model | some thing', function(hooks) {
+  setupTest(hooks);
 
-test('should correctly concat foo', function(assert) {
-  const someThing = this.subject();
-  someThing.set('foo', 'baz');
+  test('should correctly concat foo', function(assert) {
+    const someThing = run(() => this.owner.lookup('service:store').createRecord('some-thing'));
+    someThing.set('foo', 'baz');
 
-  assert.equal(someThing.get('computedFoo'), 'computed baz');
+    assert.equal(someThing.get('computedFoo'), 'computed baz');
+  });
 });
 ```
 
-See that we have used `moduleFor`, one of the several unit-test helpers provided by Ember-Qunit.
-Test helpers provide us with some conveniences, such as the `subject` function that handles lookup and instantiation for our object under test.
-Note that in a unit test you can customize the initialization of your object under test by passing to the
-`subject` function an object containing the instance variables you would like to initialize.  For example, to initialize
-the property 'foo' in our object under test, we would call `this.subject({ foo: 'bar' });`
+See that first, we are creating a new testing module using Qunit's `module` function. This will scope
+all of our tests together in one package that can be configured and run independently from other modules
+defined in our test suite.
+Also, we have used `setupTest`, one of the several unit-test helpers provided by Ember-Qunit.
```

Comment:
Should the various references to other packages (ember-qunit in this case) be links?